Showcase_Apps/VideoPlayerApp/README.md

```diff
@@ -0,0 +1,35 @@
+# Video Player App
+
```

Comment:
@robinkuck please add your (as an author) details so that you will be contacted on upon any related discussion/feedback
//Created by <your name>
//GitHub profile: <your public github profile>
//SAP Community profile: <your SAP community public profile>>